src/asana/helpers.py

```diff
@@ -350,14 +350,12 @@ def asana_comment_from_github_review(review: Review) -> str:
 
     # For each comment, prefix its text with a bracketed number that is a link to the Github comment.
     inline_comments = [
-        _wrap_in_tag("li")(
-            _wrap_in_tag("A", attrs={"href": comment.url()})(f"[{i}] ")
-            + _format_github_text_for_asana(comment.body())
-        )
+        _wrap_in_tag("A", attrs={"href": comment.url()})(f"[{i}] ")
+        + _format_github_text_for_asana(comment.body())
         for i, comment in enumerate(review.comments(), start=1)
     ]
     if inline_comments:
-        comments_html = _wrap_in_tag("ul")("".join(inline_comments))
+        comments_html = "\n".join(inline_comments)
```

Comment:
I think I'd recommend two new lines, so that there's a blank space between comments.

Reply:
I originally had it that way, but when I went to test it, it ended up with 3 newlines (this ends up with 2). I think this has to do with the fact that each inline comment itself ends in a `"\n"`, so this essentially adds the blank space for us